autoload/tagbar.vim

```diff
@@ -2413,12 +2414,24 @@ endfunction
 
 " Helper functions {{{1
 " s:AutoUpdate() {{{2
+" use timer_start to let tagbar async, this can increase vim open file performance and
+" fix windows blink when open some file.
 function! s:AutoUpdate(fname, force, ...) abort
-    call tagbar#debug#log('AutoUpdate called [' . a:fname . ']')
+    let g:tagbar_update_fname = a:fname
+    let g:tagbar_update_force = a:force
+    let g:tagbar_no_display = a:0 > 0 ? a:1 : 0
+    if has('win32') " windows use timer_start will call bug
```

Comment:
Where is this bug documented upstream (so we can track when to revert workarounds)